src/main/java/de/unistuttgart/iste/gits/reward/service/calculation/HealthScoreCalculator.java

```diff
@@ -109,7 +109,7 @@ private static List<UUID> getIds(List<Content> newDueContents) {
      * @param today          the current date
      * @return a positive number representing the health decrease
      */
-    private int calculateHealthDecrease(List<Content> newDueContents, OffsetDateTime today) {
+    public int calculateHealthDecrease(List<Content> newDueContents, OffsetDateTime today) {
```

Comment:
is it required that this is pulic?

Reply:
@PaulBredl I made it public because I can not use this  calculateHealthDecrease in test code when it is private

Reply:
But mentioned below you shouldn't use it in the test code anyway, so after that you can make it private again

Reply:
yes